db/generateData.js

```diff
@@ -0,0 +1,21 @@
+const faker = require('faker');
+
+const fakeData = [];
+
+const generate = () => {
+  for (let i = 0; i < 100; i += 1) {
+    const doc = {
+      id: i,
+      seller: faker.internet.userName(),
+      desc: faker.lorem.sentence(),
+      price: faker.commerce.price(),
+      specs: faker.lorem.sentences(),
+      shipping: 'Standard: 3-5 Business Days',
```

Comment:
This all looks wonderful.  The only thing I could possibly suggest.
```suggestion
      shipping: `Usually takes: ${Math.floor(Math.random() * 3)} to ${Math.floor(Math.random() * 5) + 3}`
```